src/haddock/libs/libalign.py

```diff
@@ -387,7 +396,29 @@ def pdb2fastadic(pdb_f):
             ("DA", "A"),
             ("DG", "G"),
             ("DC", "C"),
-            ("DT", "T"),
+            ("DT", "T"),  # 9/8/2023: adding non-standard amino-acids (https://wenmr.science.uu.nl/haddock2.4/library) # noqa: E501
```

Comment:
This `wenmr` link is already derived and for a different version of the code, please use the source: `src/haddock/cns/toppar/protein-allhdg5-4.top` related to this project

Reply:
I was not aware of this, thanks! in the new topology there's residue CHX and I have no idea what it maps to (maybe cysteine?)..do you know it?